client/src/commands/switch_impl_intf.ts

```diff
@@ -0,0 +1,49 @@
+import * as fs from "fs";
+import { LanguageClient, RequestType } from "vscode-languageclient/node";
+import { window } from "vscode";
+import { createInterfaceRequest } from "./create_interface";
+
+export const switchImplIntf = async (client: LanguageClient) => {
+  if (!client) {
+    return window.showInformationMessage("Language server not running");
+  }
+
+  const editor = window.activeTextEditor;
+
+  if (!editor) {
+    return window.showInformationMessage("No active editor");
+  }
+
+  const isIntf = editor.document.uri.path.endsWith(".resi");
```

Comment:
Should probably check that's it's actually a `.res` file as well here, if it's not `.resi`. And alert the user that this command can only run on `.res` or `.resi` files.

Reply:
It makes sense to me! I’ll add some codes to check if it is *.res either!

Reply:
https://github.com/rescript-lang/rescript-vscode/pull/404/commits/51b017c401a7355f45823448543c8f4dfea7a2c6